contrib/kube-prometheus/jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -51,4 +51,4 @@
             "version": "master"
         }
     ]
-}
```

Comment:
I think you might have an old version of `jb`, I think this might fail CI.

Reply:
the _one_ time I use vim instead of emacs ;-)